src/bridge/SequencerInbox.sol

```diff
@@ -469,11 +481,18 @@ contract SequencerInbox is GasRefundEnabled, ISequencerInbox {
     }
 
     /// @inheritdoc ISequencerInbox
-    function setIsSequencer(address addr, bool isSequencer_) external onlyRollupOwner {
+    function setIsSequencer(address addr, bool isSequencer_) external {
+        if (msg.sender != batchPosterManager) revert NotBatchPosterManager(msg.sender);
         isSequencer[addr] = isSequencer_;
```

Comment:
i think isSequencer isn't used on chain; shouldn't the batchPosterManager be calling setIsBatchPoster?

Reply:
The batchPosterManager should control both. isSequencer is indeed used off-chain to determine who should be able to sign the feed, but who should be able to sign the feed should be controlled by who should be able to post batches as the latter is a strictly more powerful ability and they should be the same entity